tests/config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Thanks, ADTKINS - these new changes look great. If the file config.json is no longer used, could you remove it from the pull request? Cheers